From: Sam Thursfield <sam.thursfield@codethink.co.uk>
Date: Mon, 17 Oct 2022 11:32:00 +0200
Subject: seccomp: Allow epoll_create1()

This is preferred over epoll_create(), and is used in GLib 2.74 and
later due to https://gitlab.gnome.org/GNOME/glib/-/merge_requests/2868

https://gitlab.gnome.org/GNOME/tracker-miners/-/issues/239
(cherry picked from commit dd0bc714666498cdbab5c87eea1def6606e2d6fe)
---
 src/libtracker-miners-common/tracker-seccomp.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libtracker-miners-common/tracker-seccomp.c b/src/libtracker-miners-common/tracker-seccomp.c
index 2f9cb81..6255e2f 100644
--- a/src/libtracker-miners-common/tracker-seccomp.c
+++ b/src/libtracker-miners-common/tracker-seccomp.c
@@ -184,6 +184,7 @@ tracker_seccomp_init (void)
 	ALLOW_RULE (pipe);
 	ALLOW_RULE (pipe2);
 	ALLOW_RULE (epoll_create);
+	ALLOW_RULE (epoll_create1);
 	ALLOW_RULE (epoll_ctl);
 	/* System */
 	ALLOW_RULE (uname);
